### PR TITLE
Update GeoWave resolvers

### PR DESCRIPTION
**Pull request by pomadchin**
## Overview

Temporary exclude GeoWave subproject from tests.

Connects https://github.com/locationtech/geotrellis/issues/2999

### CODE REVIEW

**Comment by jbouffard** [comment]
@pomadchin Do we need to update the [deploy scripts](https://github.com/locationtech/geotrellis/tree/98f4abb430027eaac69b012db439a8c635fc9f9e/.locationtech) or are they okay?

**Comment by pomadchin** [comment]
@jbouffard we don't publish it for some reason for a long time O:

**Comment by jbouffard** [comment]
@pomadchin That's interesting haha. In that case, this LGTM :+1:

File: project/Version.scala [hunk 1]
```diff
@@ -15,7 +15,7 @@
  */
 
 object Version {
-  val geotrellis  = "3.0.1-LOCAL"
+  val geotrellis  = s"3.0.1${Environment.versionSuffix}"
```
Comment (echeipesh) [inline project/Version.scala]: I think `3.0.1` is local testing leaking. We haven't published `3.0.0` yet so I think we should be fine to have this as `3.0.0-SNAPSHOT` ?

Comment (pomadchin) [reply project/Version.scala]: @echeipesh  a good point

**Pull request merged**